### DISCUSSION TITLE
KEDA for clusters with High Availability

### DISCUSSION THREAD

**Discussion by harshik9** [kedacore/keda#1877] [Q&A / Need Help]
We are trying to implement Active-Active production clusters in different regions. Is there a way we can implement KEDA for Active - Active clusters. How can we achieve this?

**Comment by tomkerkhove** [comment]
We have documentation on high-availability but basically we can only run 1 instance due to upstream limitations - https://keda.sh/docs/2.3/operate/cluster/#high-availability

I haven't heard about setting up active-active clusters with KEDA but am interested in the scenario. Do you want to mitigate the SO's from cluster A to cluster B or what are you looking for? Because that seems more like a general Kubernetes question rather than Kubernetes.

You may find https://github.com/kedacore/keda/issues/1587 interesting
